themes/book/app/routes/$.tsx

```diff
@@ -96,9 +96,9 @@ export default function Page() {
   const { container, outline } = useOutlineHeight();
   const top = useThemeTop();
   const article = useLoaderData<PageLoader>() as PageLoader;
-  const pageDesign: BookThemeTemplateOptions = (article.frontmatter as any)?.design ?? {};
+  const pageDesign: BookThemeTemplateOptions = (article.frontmatter as any)?.options?.design ?? {};
   const siteDesign: BookThemeTemplateOptions =
-    (useSiteManifest() as SiteManifest & BookThemeTemplateOptions) ?? {};
```

Comment:
I would have expected this to be `(...).design` - maybe there is something else going on here that isn't just simple frontmatter reading.